packages/plugin-block-charts/src/client/chart/antd/GroupedTable.ts

```
@@ -91,15 +90,19 @@ export class GroupedTable extends AntdChart {
             minimumFractionDigits: number,
             maximumFractionDigits: number,
           };
+
           const numberFormat = new Intl.NumberFormat('zh-CN', options);
           const num = String(value[dataValue.key]).includes(',')
             ? String(value[dataValue.key]).replace(/,/g, '')
             : value[dataValue.key];
+
           if (!isNaN(num)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)